src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1701,10 +1707,27 @@ public function copyLanguage(
         }
 
         $structure = $this->load($uuid, $webspaceKey, $srcLanguageCode);
+        $parentNode = $this->getSession()->getNodeByIdentifier($structure->getUuid())->getParent();
+        $resourceLocator = $this->getResourceLocator();
 
         $data = $structure->toArray(true);
+        foreach ($destLanguageCodes as $destLanguageCode) {
+            if ($structure->hasTag('sulu.rlp')) {
+                $parentUrl = $resourceLocator->getResourceLocatorByUuid(
+                    $parentNode->getIdentifier(),
+                    $webspaceKey,
+                    $destLanguageCode
+                );
+                $rlp = $this->getResourceLocator()->getStrategy()->generate(
+                    $structure->getPropertyValue('title'),
```

Comment:
because the "cached" is wrong at this place ... it should be recreated with the newest title and parent url for the new language!